docs/proposals/instrumentation-of-services.md

```diff
@@ -0,0 +1,70 @@
+# Best practices for instrumenting infrastructure components in OpenShift
+
+All OpenShift components MUST expose the following endpoints on either their public HTTPS port, or a public HTTP port if none exists
```

Comment:
These are already the kubernetes and openshift conventions (they are what every component in the core uses).  The other section covers what happens if you're adapting something else.